dev/com.ibm.ws.security.spnego_fat/bnd.bnd

```diff
@@ -0,0 +1,38 @@
+#*******************************************************************************
+# Copyright (c) 2020 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+#*******************************************************************************
+-include= ~../cnf/resources/bnd/bundle.props
+bVersion=1.0
+
+src: \
+    fat/src
+
+fat.project: true
+
+tested.features: pages-3.0
```

Comment:
I didn't se repeats so no jakarta 9 testing from what I could tell.  This can be removed?

Reply:
This FAT does have one FVT run with EE9 - ServerPrincipalNamesTest.